addons/default/visiosoft/advs-module/src/Http/Controller/AdvsController.php

```diff
@@ -360,6 +360,38 @@ public function index($category = null, $city = null)
             ];
         }
 
+        if (($cities = \request()->city) && $cities = $cities[0]) {
```

Comment:
maybe smarter code then this? @diashalabi

Reply:
I don't think so